docs/getting-started/system-requirements.md

```diff
@@ -0,0 +1,14 @@
+
+# System requirements
+
```

Comment:
This is done by most if not all OSs automatically and the swap file is scaled according to needs.

Can you elaborate why this is needed?